neuvector-enforcer.yaml

```diff
@@ -0,0 +1,55 @@
+package:
+  name: neuvector-enforcer
+  version: 5.3.2
+  epoch: 0
+  description: "NeuVector Enforcer"
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - busybox
+      - ca-certificates
+      - consul
+      - ethtool
+      - iproute2
+      - iptables
+      - jq
+      - neuvector-agent
+      - neuvector-dp
+      - neuvector-monitor
+      - neuvector-nstools
+      - yq
+
+environment:
+  contents:
+    packages:
+      - busybox
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: edbdcba632835d56dcc92ba86323c8a196471289
+      repository: https://github.com/neuvector/neuvector.git
+      tag: v${{package.version}}
+
+  - runs: |
+      mkdir -p ${{targets.contextdir}}/etc
+      install -Dm755 scripts/sysctl.conf ${{targets.contextdir}}/etc/
+
+      mkdir -p ${{targets.contextdir}}/usr/local/bin
+      install -Dm755 scripts/configure.sh ${{targets.contextdir}}/usr/local/bin/
+      install -Dm755 scripts/teardown.sh ${{targets.contextdir}}/usr/local/bin/
+      install -Dm755 scripts/runtime-gdb.py ${{targets.contextdir}}/usr/local/bin/
+
+      ln -sf /usr/bin/yq ${{targets.contextdir}}/usr/local/bin/yq
```

Comment:
do you think that that'd be better if we could create a -compat package for this?

Reply:
Nope, NeuVector checks for these paths directly and is absolutely useless otherwise. Let's just give it what it wants